dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -562,6 +563,8 @@ private Workflow createOrGetWorkflow(Class workflowType, String repository, User
                 throw new CustomWebApplicationException("User does not have an account on Dockstore.", LAMBDA_FAILURE);
             }
 
+            StringInputValidationHelper.checkEntryName(workflowType, workflowName);
```

Comment:
If I'm reading this correctly the entry name will be checked only during create. One of my questions coming to the PR was what would happen to existing workflows with names that don't pass validation (exceed length or contain spaces/special characters). I presume there would be no effect?

Reply:
Yeah, that's correct. This validation has no effects on existing workflows with invalid names